packages/gestalt/src/Button.flowtest.js

```diff
@@ -4,8 +4,10 @@ import Button from './Button.js';
 
 const Valid = <Button text="Next" />;
 
-// $FlowExpectedError[prop-missing]
```

Comment:
@AlbertCarreras could we add a couple more tests of buttons that are correct and incorrect to test the flow types?